test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/Dockerfile

```diff
@@ -9,7 +9,7 @@ FROM ubuntu AS w3c
 #Install git
 WORKDIR /w3c
 RUN apt-get update && apt-get install -y git
-RUN git clone https://github.com/w3c/trace-context.git
+RUN git clone --branch level-1 https://github.com/w3c/trace-context.git
```

Comment:
> There are 2 versions of the specification:

> [W3C Recommendation](https://www.w3.org/standards/types#REC) 23 November 2021, Level 1: https://www.w3.org/TR/trace-context/#combined-header-value release
[W3C Candidate Recommendation Snapshot](https://www.w3.org/standards/types#CR) 18 April 2023, Level 2: https://www.w3.org/TR/trace-context-2/#combined-header-value

@Kielek How do you determine the latest stable release of W3C? How did you find out that level 1 is the latest stable and level 2 is rc?

Reply:
Please check the headers of the pages. Statuses are pretty clear.

If we speaking about checking which branch to take. I have decided based on the commits date (in sync what is in page headers) and its names.